homeassistant/components/jewish_calendar/quality_scale.yaml

```diff
@@ -0,0 +1,92 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
I would recommend to reuse `result` instead of doing `result2`

Reply:
Done